src/services/doppler-legacy-client.ts

```diff
@@ -566,7 +566,7 @@ export class HttpDopplerLegacyClient implements DopplerLegacyClient {
   }
 
   public async getUserData() {
-    const response = await this.axios.get('/WebApp/GetUserData');
+    const response = await this.axios.get('/WebApp/GetUserData', { timeout: 60000 });
```

Comment:
Is not it too much to wait for a minute?

Reply:
This timeout works only in the worst case, think of  a deploy in which the server does not return any response ever. Or if some user has very slow connection we should allow it right? Chrome has 5 minutes to timeout a connection.  What do you suggest?

Reply:
![image](https://user-images.githubusercontent.com/2439363/106502609-5a28c400-64a3-11eb-99c7-0f86a048469f.png)

Reply:
Does anybody will wait one minute till receive a response? 🤔 
I think that Andres is right, maybe the timeout could be... 5 seconds?

Reply:
In the worst case scenario only, that is no response from server or excesive slow connection. In normal everyday this number does not affect. I've put this timeout in 30 seconds.